src/Http/Response.php

```diff
@@ -161,6 +161,8 @@ class Response implements ResponseInterface
         'gz' => 'application/x-gzip',
         'bz2' => 'application/x-bzip',
         '7z' => 'application/x-7z-compressed',
+        'hal+json' => ['application/hal+json', 'application/vnd.hal+json'],
```

Comment:
Wouldn't this just be `hal` ?

Reply:
There is a hal+xml specification too. How else should these be distinguished?

Reply:
I never see `+` in a type name, but there doesn't seem to be any other name for hal objects. *shrug*

Reply:
Agreed, it's goofy. I am fine dropping hal+xml al together and just going with `'hal' => ['application/hal+json', 'application/vnd.hal+json']`.